src/embed/world-renderer.js

```diff
@@ -168,13 +169,37 @@ WorldRenderer.prototype.submitFrame = function() {
     this.vrDisplay.submitFrame();
   }
 };
-
+WorldRenderer.prototype.dispose = function() {
+    var eyeLeft = this.scene.getObjectByName('eyeLeft'),
+        eyeRight = this.scene.getObjectByName('eyeRight');
+    if(!eyeLeft || !eyeRight) return
```

Comment:
We shall definitely put [ESLint](http://eslint.org/) with [Google rules](https://github.com/google/eslint-config-google) with `pre-commit` hook to avoid such contribution problems in the future.